Aztec/Classes/Extensions/UIFont+Traits.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import UIKit
+
+
+// MARK:- UIFont Traits Helpers
+//
+extension UIFont {
+
+    /// Returns a new instance of the current font with its SymbolicTraits Updated.
+    ///
+    /// - Parameters:
+    ///     - trait: Trait to be updated
```

Comment:
Since the `trait` parameter is a mask which can potentially hold several traits, it should be named `traits` (plural) instead.